src/routes/helpers/transformers.js

```diff
@@ -0,0 +1,76 @@
+import SchemaHelper from '../../helpers/SchemaHelper'
+import { warning } from '../../utils/logger'
+
+// Used scopes dict
+const scopedContexts = {
+  dc: 'http://purl.org/dc/elements/1.1/',
+  prov: 'https://www.w3.org/TR/prov-o/#',
+  skos: 'http://www.w3.org/2004/02/skos/core#',
+  rdf: 'https://www.w3.org/2000/01/rdf-schema'
+}
+
+/**
+ * Transform document to a JSON-LD structured document
+ * @param {string} type
+ * @param {Object} data
+ * @returns {Object} JSON-LD structured document
+ */
+export const transformJsonLD = (type, data) => {
+  const schemaHelper = new SchemaHelper()
+  const schemaTypes = schemaHelper.getTypeDescription(type).split(',')
+  const prefixes = Object.keys(scopedContexts)
+
+  // Base JSON-LD document
+  const jsonLdData = {
+    '@context': [
+      'https://schema.org/',
+      scopedContexts
+    ],
+    '@type': schemaTypes
+  }
+
+  // Iterate all keys in the data document
+  Object.keys(data).forEach(key => {
+    // Get the property for the current key
+    const property = schemaHelper.findPropertyType(type, key)
+
+    // Use the current value by default
+    let elementValue = data[key]
+
+    // Transform the value when it's a relational property
+    if (elementValue && schemaHelper.isRelationalProperty(property)) {
```

Comment:
it would be nice to format dates in iso8601 format, would this be the right place to put a specific type conversion?

Reply:
@ChristiaanScheermeijer Can we have feedback on this point? I can add the conversion if needed, I just need to know if this is the right place to do it.

Reply:
@alastair this is the point you would check for date types indeed. I have to say though, I found myself a bit struggling with the Neo4jDate(Time) types that are being used. Looking forward to seeing your solution regarding this.

Reply:
see here: https://github.com/neo4j/neo4j-javascript-driver#numbers-and-the-integer-type
There are 2 options that we can choose - one is to add the `disableLosslessIntegers` option when connecting to neo4j, which will only allow us to support js integers (56 bit instead of 64 bit), or we can convert the numbers as we read them with the toNumber() method on the integer object. Do you have a preference? perhaps the losslessintegers config option is easiest for us at the moment, I'm not sure I can think of a situation at the moment where we need 64 bit integers

Reply:
the js driver for neo4j already has the functionality to convert these dates, done in 4d9817d

Reply:
are we encountering any issues with this `int` to `float` conversion?

Reply:
I don't think we currently have any integers in the CE which are too large, I think we can open this as a separate ticket and follow it up at a later stage